src/javascript/core/user.js

```diff
@@ -22,12 +22,17 @@ const Store = require('./store');
  * Init
  *
  * @param {String} value The value of a userID to use if one is not stored
+ * @param {String} cookieDomain The domain that should be used to cookie te user
  * @return {String} - The user ID if present, or a generated UID if not
  */
-function init(value) {
-	store = new Store(defaultUserConfig.name, defaultUserConfig);
+function init(value, cookieDomain) {
+	if ( cookieDomain ) {
+		defaultUserConfig.domain = cookieDomain;
```

Comment:
Is there a reason to manually pass the domain vs. calling `window.location.hostname`?
To hopefully answer my own question: Because `www.ft.com` would be unhelpful?

Reply:
Sorry I missed the point, and this https://github.com/Financial-Times/o-tracking/blob/fcbb8b62bb80350b9c815f60bcc47bf8371548f2/src/javascript/core/user.js#L15

Reply:
Yep, so this change is really for o-tracking usage outside of ft.com, in scout.asia for example where this default isn't desirable.